scripts/localization.swift

```diff
@@ -30,7 +30,7 @@ fileprivate var twnTokenRegex: NSRegularExpression? = {
 
```

Comment:
The important changes are here - most of the rest of the diff is due to changes in the localized output